did-web/src/lib.rs

```diff
@@ -91,7 +91,14 @@ impl DIDResolver for DIDWeb {
             Ok(url) => url,
         };
         // TODO: https://w3c-ccg.github.io/did-method-web/#in-transit-security
-        let client = match reqwest::Client::builder().build() {
+
+        let mut headers = reqwest::header::HeaderMap::new();
+        headers.insert(
+            "User-Agent",
+            reqwest::header::HeaderValue::from_static("ssi/0.2"),
+        );
```

Comment:
It would be nice if this could be generated from crate metadata or something, with the exact version number.
The example in https://docs.rs/reqwest/0.11.3/reqwest/struct.ClientBuilder.html#example shows how to get the crate name and version. But here we are in the `did-web` crate, not the `ssi` crate, so that is not readily available. To get `ssi`'s exact version I think `ssi` would need to export that e.g. as a public constant. Or the [built](https://docs.rs/built/0.5.1/built/) crate could be used as a build-time dependency to get it.
It could be good to include higher-level applications and their versions as well, e.g. `didkit`; for that, the `DIDWeb` struct could be changed to add an option to override the user-agent so the application (e.g. `didkit` or `didkit-cli`) could set it. Maybe the `did-web` should exports its version too?

Anyway, I think this change is good, but just noting these considerations here. We will need to update `0.2` when the `ssi` minor version changes.

Reply:
Just to suggest more options:
- the user agent could be part of the interface (in the struct for did-web?); or
- `ssi` could have a global reqwest client.

Reply:
But +1 for using `built`.

Reply:
Got it. I'm creating a constant in lib.rs in SSI and importing the constant whenever needed.